yt_dlp/extractor/lecturio.py

```diff
@@ -58,7 +58,7 @@ class LecturioIE(LecturioBaseIE):
                     https://
                         (?:
                             app\.lecturio\.com/([^/]+/(?P<nt>[^/?#&]+)\.lecture|(?:\#/)?lecture/c/\d+/(?P<id>\d+))|
-                            (?:www\.)?lecturio\.de/[^/]+/(?P<nt_de>[^/?#&]+)\.vortrag
+                            (?:www\.)?lecturio\.de/(?:[^/]+/){1,2}(?P<nt_de>[^/?#&]+)\.vortrag
```

Comment:
Since the `.vortrag` always marks the segment we want, we could make it more relaxed

```suggestion
                            (?:www\.)?lecturio\.de/(?:[^/]+/)+(?P<nt_de>[^/?#&]+)\.vortrag
```